internal/view/app.go

```diff
@@ -413,7 +413,7 @@ func (a *App) switchContext(name string, loadPods bool) error {
 			return e
 		}
 		v := a.Config.ActiveView()
```

Comment:
Actually we no longer need to track v here and we can set the active view using the config solely ie 
```go
if a.Config.ActiveView() == "" || ... {
   a.Config.SetActiveView("pod")
}
...
```